test/integration/tags/for_tag_test.rb

```diff
@@ -23,16 +23,16 @@ def test_for
   yo
 
 HERE
```

Comment:
should we fix the indentation here too (to use `<<~`)?

Reply:
Yeah however I might bring this in with the next round of changes. Hopefully there is a rubocop rule I can use to suggest using <<~ over <<